### PR TITLE
[Theming] Use color names in props

### PR DESCRIPTION
**Pull request by zachstence**
### Description

Enables the usage of `color`/`colorPalette`/`colorScale` names in props

E.g.

```svelte
<Story name="Using color token">
	<Tabs color="accent">
		<Tab label="Tab 1" id="tab1">Tab 1 content</Tab>
		<Tab label="Tab 2" id="tab2">Tab 2 content</Tab>
		<Tab label="Tab 3" id="tab3">Tab 3 content</Tab>
	</Tabs>
</Story>
```

```svelte
<Story
	name="With seriesOrder and seriesColors"
	args={{
		x: 'x',
		y: 'y',
		series: 'series',
		seriesOrder: ['ivory', 'blue', 'violet', 'olive'],
		seriesColors: { ivory: 'primary', blue: 'secondary', violet: 'accent', olive: 'negative' }
	}}
/>
```

### Checklist

- ~~[ ] For UI or styling changes, I have added a screenshot or gif showing before & after~~
- ~~[ ] I have added a [changeset](https://github.com/evidence-dev/evidence/blob/main/CONTRIBUTING.md#adding-a-changeset)~~
- ~~[ ] I have added to the docs where applicable~~
- ~~[ ] I have added to the [VS Code extension](https://github.com/evidence-dev/evidence-vscode) where applicable~~

### CODE REVIEW

**Comment by zachstence** [comment]
Main thing left is figuring out what to do with the existing color aliasing in `Column` and `Reference*` components

File: packages/ui/core-components/src/lib/unsorted/ui/QueryViewerSupport/prismtheme.css [hunk 1]
```diff
@@ -88,7 +88,7 @@ pre[class*='language-'] {
 }
 
 .token.comment {
-	color: var(--blue-700);
+	color: #1d4ed8;
```
Comment (ItsMeBrianD) [inline packages/ui/core-components/src/lib/unsorted/ui/QueryViewerSupport/prismtheme.css:91]: Was this meant to be hard coded?

File: packages/ui/core-components/src/lib/unsorted/viz/bubble/BubbleChart.svelte [hunk 2]
```diff
@@ -36,9 +40,16 @@
 	export let yMax = undefined;
 
 	export let shape = undefined;
+
 	export let fillColor = undefined;
+	$: fillColorStore = resolveColor(fillColor);
+	$: console.log({ $fillColorStore });
```
Comment (ItsMeBrianD) [inline packages/ui/core-components/src/lib/unsorted/viz/bubble/BubbleChart.svelte:46]: console log

File: sites/example-project/src/pages/text-and-metrics/code-blocks/+page.md [hunk 3]
```diff
@@ -54,7 +54,7 @@ z = x + y
 ```css
 pre {
 	overflow: scroll;
-	background: var(--grey-800);
+	background: #1f2937;
```
Comment (ItsMeBrianD) [inline sites/example-project/src/pages/text-and-metrics/code-blocks/+page.md:57]: Same as above

File: sites/example-project/src/pages/text-and-metrics/markdown/+page.md [hunk 4]
```diff
@@ -116,7 +116,7 @@ z = x + y
 ```css
 pre {
 	overflow: scroll;
-	background: var(--grey-800);
+	background: #1f2937;
```
Comment (ItsMeBrianD) [inline sites/example-project/src/pages/text-and-metrics/markdown/+page.md:119]: Same as above

**Review by ItsMeBrianD — CHANGES_REQUESTED** [review]

File: packages/ui/core-components/src/lib/unsorted/ui/QueryViewerSupport/prismtheme.css (hunk 1 above)
Comment (zachstence) [reply packages/ui/core-components/src/lib/unsorted/ui/QueryViewerSupport/prismtheme.css:91]: Yes. I removed the CSS color vars and left this here to maintain our prism theme

File: sites/example-project/src/pages/text-and-metrics/code-blocks/+page.md (hunk 3 above)
Comment (zachstence) [reply sites/example-project/src/pages/text-and-metrics/code-blocks/+page.md:57]: Yes

File: packages/ui/core-components/src/lib/unsorted/viz/bubble/BubbleChart.svelte (hunk 2 above)
Comment (zachstence) [reply packages/ui/core-components/src/lib/unsorted/viz/bubble/BubbleChart.svelte]: [f982fa3](https://github.com/evidence-dev/evidence/pull/2776/commits/f982fa334cc9891cf113b4669d82ce255a2ba42e)

**Review by ItsMeBrianD — APPROVED** [review]

**Pull request merged**